src/async_fuse/memfs/fs_util.rs

```diff
@@ -273,6 +273,35 @@ impl Default for FileAttr {
     }
 }
 
+impl From<FileAttr> for FuseAttr {
+    fn from(attr: FileAttr) -> Self {
+        let (a_time_secs, a_time_nanos) = time_from_system_time(&attr.atime);
+        let (m_time_secs, m_time_nanos) = time_from_system_time(&attr.mtime);
+        let (c_time_secs, c_time_nanos) = time_from_system_time(&attr.ctime);
+
+        FuseAttr {
+            ino: attr.ino,
+            size: attr.size,
+            blocks: attr.blocks,
+            atime: a_time_secs,
+            mtime: m_time_secs,
+            ctime: c_time_secs,
+            atimensec: a_time_nanos,
+            mtimensec: m_time_nanos,
+            ctimensec: c_time_nanos,
+            mode: crate::async_fuse::util::mode_from_kind_and_perm(attr.kind, attr.perm),
```

Comment:
Suggest replacing with `util::mode_from_kind_and_perm`.

Reply:
Thank you for your feedback. Since it already imports module `use crate::common::util`, I don't think this will work directly. Could you please make some suggestions?

Reply:
You can import this crate at the top of this file `use xxx::util as fuse_util` or directly import this help function.

Reply:
OK, I have fixed it. Please review the changes.